.github/workflows/release.yml

```diff
@@ -0,0 +1,41 @@
+name: Build & Release
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build_and_release:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: '3.10'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install poetry
+    - name: Build package
+      run: |
+        poetry self add "poetry-dynamic-versioning[plugin]"
+        poetry config repositories.testpypi https://test.pypi.org/legacy/
```

Comment:
What is the purpose of always publishing to both repositories?

Reply:
If it fails on test it won't publish to the real pypi 🙂 We don't _have_ to though, it was just an extra safety thing when we were first trying out pdv.

Reply:
Removed it ✅ I agree its pointless at this stage 👍